server/build.gradle

```diff
@@ -25,9 +28,6 @@ repositories {
 
 // External dependencies that our application utilizes
 dependencies {
-  // Google core libraries for Java, various useful utilities
```

Comment:
We were only using the `ImmutableMap.of()` method from Guava, and that functionality is now provided by `java.util.Map.of()`. So I replaced all the uses of `ImmutableMap.of()` and removed this dependency.